src/aws-compile-servicecatalog.js

```diff
@@ -149,10 +150,20 @@ class AwsCompileServiceCatalog {
       }
       newFunction.Properties.ProvisioningParameters[index].Value = value;
     };
+
+    const removeProvisioningParamValue = (key) => {
+      if (!key) {
+        return;
+      }
+
+      const index = newFunction.Properties.ProvisioningParameters
+        .findIndex(kv => kv.Key === key);
+      if (index === -1) {
+        return;
+      }
+      newFunction.Properties.ProvisioningParameters[index] = {};
```

Comment:
Should we just splice out this whole index? Or is there some value to making it set to an empty object?

Reply:
if you remove it it will inject the default value. (that was my first attempt!)